cli/cmd/sources.go

```diff
@@ -372,6 +538,19 @@ func init() {
 	sourcesCmd.AddCommand(sourceUpdateCmd)
 	sourcesCmd.AddCommand(sourceStatusCmd)
 
+	sourcesCmd.AddCommand(sourceEnableCmd)
+	sourceEnableCmd.PersistentFlags().StringVarP(&sourceNamespaceFlag, namespaceFlagName, "n", "default", "Kubernetes Namespace for Source")
```

Comment:
will we see the `-n` also for `odigos sources enable namespace`? if I understand correctly it is not used for this subcommand.

do you think we can add it only to the relevant commands to reduce chance of mistakes?